Tests/FunctionalTests/MiscellaneousTests.swift

```diff
@@ -213,6 +213,9 @@ class MiscellaneousTestCase: XCTestCase {
     }
 
     func testSwiftTestParallel() throws {
+        // <rdar://problem/69448176> Fix and re-enable test related to "ParallelTestsPkg"
+        try XCTSkip()
```

Comment:
These `try XCTSkip()` lines are generating the following errors in Xcode 12:

> Result of 'XCTSkip' initializer is unused

> No calls to throwing functions occur within 'try' expression

Should these instead be `throw XCTSkip()`?